Makefile

```diff
@@ -2,6 +2,8 @@
 ZARF_BIN := ./build/zarf
 UNAME_S := $(shell uname -s)
 UNAME_P := $(shell uname -p)
+# Need a clean way to map this, arch and uname -a return x86_64 for amd64
+ARCH := amd64
```

Comment:
Should we be checking if the arch should be set to arm somewhere around here?

Reply:
Yeah we could, but there's no clean way to do it that I've seen.  For ci it doesn't matter since we don't have arm VMs right now, but for local arm testing it would be a problem.  Also I really hate makefile variables in general, so that's a thing.